api/openapi-spec/swagger.json

```diff
@@ -12071,6 +12071,16 @@
       "core_v1"
      ],
      "operationId": "proxyCoreV1PUTNamespacedPod",
+     "parameters": [
+      {
+       "name": "body",
+       "in": "body",
+       "required": true,
+       "schema": {
+        "$ref": "#/definitions/io.k8s.api.core.v1.Pod"
```

Comment:
This isn't accurate. The body sent to a pod proxy endpoint has nothing to do with a Pod API definition, it completely depends on what the pod expects

Reply:
This applies to all of these. The proxy endpoints and subresources for pods and services have no schema info they can provide

Reply:
OK. I will close this PR and investigate this more.